### ISSUE TITLE
DSF File Support?

### ISSUE THREAD

**Issue by erkkav** [cybercase/webminidisc#55]
Any chance for a DSF file support? At the moment file browser doesn't recognise those files at all. DSF files work in Platinum-MD so I think it should be possible feature.

**Comment by cybercase** [comment]
Yes, that's doable. I'll try to add support for DSF files when I update the ffmpeg library.

**Comment by erkkav** [comment]
Thanks a lot. I think it's the only feature I would need to make it near perfect in my use case.
